rest-client/client-common/src/main/java/io/apicurio/registry/rest/client/impl/RegistryClientImpl.java

```diff
@@ -17,10 +17,8 @@
 package io.apicurio.registry.rest.client.impl;
 
 import java.io.InputStream;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
uh oh!

Reply:
That is IDEA's work, should I remove it?

Reply:
I believe yes. Unless you want to import the entire package

Reply:
Yes, please, revert this. You can configure this in IDEA e.g. I have 99 as the limit so please try to avoid wildcard imports.

Reply:
Done.